lib/eco/eco.py

```diff
@@ -466,6 +450,8 @@ def change(self, index):
             self.ui.textEdit.setText(text)
         elif index == "Default":
             text = self.tm.export("/dev/null")
+            if text is None:
+                text = ""
```

Comment:
Are you sure this can be deleted? I think if export fails it returns None and setText might not work with None as the argument.